src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -3489,5 +3488,89 @@ GUI_TEST_CLASS_DEFINITION(test_7715) {
     GTUtilsLog::checkContainsMessage(os, ltSizeNameList, false);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7720) {
+    GTFileDialog::openFile(os, testDir + "_common_data/realign_sequences_in_alignment/", "amino_ext.aln");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    GTUtilsMsaEditor::setMultilineMode(os, false);
+
+    auto activeWindow = GTUtilsMsaEditor::getEditor(os)->getUI()->getUI(0);
+    CHECK_SET_ERR(activeWindow != nullptr, "Sequence widget is visible");
+
+    auto wgt = GTUtilsMSAEditorSequenceArea::getSequenceArea(os, 0);
+    CHECK_SET_ERR(wgt != nullptr, "Sequence area is visible");
+
+    auto splitter = GTWidget::findSplitter(os, "name_and_sequence_areas_splitter", activeWindow);
+    CHECK_SET_ERR(splitter != nullptr, "Splitter is visible");
+
+    QSplitterHandle* handle = splitter->handle(1);
+    CHECK_SET_ERR(handle != nullptr, "MSA Splitter handle is NULL");
+
+    GTWidget::click(os, handle);
+    QPoint p = GTMouseDriver::getMousePosition();
+    int deltaMove = wgt->width() / 3 * 2;
+    p.setX(p.x() + deltaMove);
+    GTMouseDriver::press();
+    GTMouseDriver::moveTo(p);
+    GTMouseDriver::release();
+
+    GTThread::waitForMainThread();
+    GTUtilsMsaEditor::setMultilineMode(os, true);
+
+    auto count = GTUtilsMsaEditor::getEditor(os)->getUI()->getChildrenCount();
+    bool allAreVisible = true;
+    for (uint i = 0; i < count; i++) {
```

Comment:
Do not use unsigned types unless it is required by some external library.
Unsigned types introduce warnings and unexpected behaviors when used in the same expressions with signed types.

Reply:
<MaEditorMultilineWgt.h>:

    // Return lines count in multiline widget
    uint getChildrenCount() const {
        return uiChildCount;
    }

Reply:
already fixed

Reply:
I had the same comment to the original big patch: all unsigned types you introduced should go away

Reply:
Let's change uint -> int in separate issue/PR ?
Too many places to change - the this PR will be messed

Reply:
Please fix only new uints introduced in this PR. AFAIU all other uints in UGENE were already cleaned up.